code-challenges.test.js

```diff
@@ -15,24 +15,72 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+describe("shuffle", () => {
+    const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+    const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+    it("takes in an array, removes the first item from the array and shuffles the remaining items.", () => {
+        expect(shuffle(colors1)).not.toEqual(
+            expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+        expect(shuffle(colors2)).not.toEqual(
+            expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+    })
+  })
+  //ReferenceError: shuffle is not defined
+
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+//create a function shuffle that takes in an array
+//remove first item form that array use shift method
+//shuffle remaining items using Fisher-Yates Algo
+//return the shuffled array
+
+//--------------------------------------
+//This is interesting the code passes eventhough the array isn't being shuffled. My test may not be set up right.
```

Comment:
Your test is SUPER close!  You have the .not matcher included which will negate the .toEqual matcher.  Removing the .not should fix this.